applications/dairdrop.md

```diff
@@ -0,0 +1,232 @@
+
+  This airdrop application will serve projects within the Polkadot ecosystem that have a common airdrop need, as well as new teams creating projects on Polkadot in the future. It will first be utilized in the Dota 2.0 network. Continuous token airdrop activities are especially important for purely community-driven projects, and this application will be frequently used by these projects.
+
+
+- Are there any projects similar to yours in related ecosystems? 
+    Many public chains have dedicated applications for airdrops, which facilitate the monitoring of on-chain airdrop activities. However, these airdrop applications are often specific to certain chains and have a certain user threshold, making them more suitable for technical teams rather than general users. Our goal is to create an airdrop tool that can be used by all projects within the Polkadot ecosystem.
+
+## Team :busts_in_silhouette:
+
+<!-- > Please note that the data provided in this section is for administrative and informational purposes only. All beneficiaries of a grant must also be listed in the KYC/KYB process during the application phase. See our [FAQ](https://grants.web3.foundation/docs/faq#what-is-kyckyb-and-why-do-i-have-to-provide-this-data) for more info. -->
+
+### Team members
+
+- Weaver
+- MockBL
+
+### Contact
+
+- **Contact Name:** [Weaver](https://x.com/weaver380978)
+- **Contact Email:** liny153537@gmail.com
+<!-- - **Website:** Your website -->
+
+### Legal Structure
+
+- **Registered Address:** No. 601, Labor Building, Jinhui Road, Guandu District, Kunming City, Yunnan Province, China.
+<!-- - **Registered Legal Entity:** Name of your registered legal entity, if available. (e.g. Duo Ltd.) -->
+
+### Team's experience
+
+- weaver 
+  Five years of development experience with Polkadot, core code contributor in the Dota community, and involved in the development of multiple parachains. Possesses extensive experience in blockchain product development within the Polkadot and Ethereum ecosystems.
+  
+- MockBL
+  13 years of experience in front-end development and UI design.
+
+
+### Team Code Repos
+
+- https://github.com/dota-network/dairdrop
+- https://github.com/dota-network/dota-super-chain
+- https://github.com/dota-network/dota2.0-website
+
+<!-- Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine. -->
+
+- `weaver` https://github.com/jyaymusk
+- `MockBL` https://github.com/MockBL
+
+<!-- ### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/{person_1}
+- https://www.linkedin.com/{person_2} -->
+
+
+## Development Status :open_book:
+
+Under development.
+
+- https://github.com/dota-network/dairdrop
+- https://github.com/dota-network/dota-super-chain
```

Comment:
Is there nothing but a readme to this project?

Reply:
> Is there nothing but a readme to this project?

The Dota Super Chain has completed most of the code, developed by community contributors, and  will be merged soon. The dairdrop is currently being prepared for development.

Reply:
where can I find the code for dota-super-chain?

Reply:
> where can I find the code for dota-super-chain?

Our original plan for the Dota community was to develop an EVM-compatible chain with NPOS, but there were no existing technology templates, and development would take time(even though we already have very experienced Substrate developers). Fortunately, Chainsupport reached out to us, offering to contribute code for this purpose, as they are collaborating with several different teams that have similar needs. They are currently focused on developing the public part of the code and expect to complete it within a month. After that, we will use their template to develop the Dota 2.0 network. Additionally, with the support of our community developers, we will be able to help them solve any technical challenges they encounter during the development process, as we have more experience with Substrate technology.

code: [https://github.com/ChainSupport/frontier-pos-template](https://github.com/ChainSupport/frontier-pos-template)